src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -593,7 +593,7 @@ public string GetReadme()
             return node?.Value;
         }
 
-        private static bool? AttributeAsNullableBool(XElement element, string attributeName)
```

Comment:
Client repo has [`CA1822: Mark members as static`](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1822) analyzer currently turned on as `Info` but will be changed `warning` soon. Since `AttributeAsNullableBool` method doesn't access instance data, can we keep it as `static` please.

![image](https://user-images.githubusercontent.com/52756182/112510522-ba730e00-8d4e-11eb-8f46-daed6438284b.png)

Reply:
`GetIdentity()` is an instance method. It is now called in this method to fix the exception message.

Reply:
I see, I missed that part. Thanks Joel.